doc/man1/openssl-ca.pod.in

```diff
@@ -285,11 +285,25 @@ used).
 =item B<-subj> I<arg>
 
 Supersedes subject name given in the request.
+
 The arg must be formatted as C</type0=value0/type1=value1/type2=...>.
-Keyword characters may be escaped by C<\> (backslash), and whitespace is
-retained.
+Special characters may be escaped by C<\> (backslash), whitespace is retained.
 Empty values are permitted, but the corresponding type will not be included
 in the resulting certificate.
+Giving a single C</> will lead to an empty sequence of RDNs (a NULL-DN).
+If the B<-multivalue-rdn> option is used then multi-valued AVAs can be formed
+by placing a I<+> character instead of a I</> before subsequent type names.
+
+=item B<-multivalue-rdn>
+
+This option causes the -subj argument to be interpreted with full
+support for multi-valued RDNs for the B<-subj> argument
```

Comment:
`-subj argument` is duplicated once without emphasis, should be sufficient to mention once. Also missing period at the end of sentence.

Reply:
Oops, fixed.